### ISSUE TITLE
[Bug] Pan Mode is not displayed on Graph Canvas Menu

### ISSUE THREAD

**Issue by LukeG89** [chrisgoringe/cg-controller#325]
After the implementation of [this feature request](https://github.com/chrisgoringe/cg-controller/issues/297), the icon and label on Graph Canvas Menu don't change to "Pan Mode", it keeps "Select mode" even though we are actually in Pan Mode.

https://github.com/user-attachments/assets/f813f0e0-13bb-4307-8799-9bf22dfc323e

**Comment by chrisgoringe** [comment]
Thx. Please check console for errors? (I'll probably always ask that!)

**Comment by LukeG89** [comment]
No errors, only these messages when clicking:

```
Update all controllers requested because 'read_only'. Controller refused with no retry
debug.js:10 Update all controllers request 'read_only' cancelled because delay was negative
```

**Comment by chrisgoringe** [comment]
fixed on 1.4
